docs/SpaceTemplates.rst

```diff
@@ -108,11 +109,16 @@ In which:
 +-------------------------+------------------------------+----------------------------------------------------------------------------+
 | **bannerPath**          | ``string``                   | The path of the space banner file inside your extension war.               |
 +-------------------------+------------------------------+----------------------------------------------------------------------------+
-| **homePageApplication** | ``SpaceApplication``         | The application to use in the home page of a space.                    |
+| **permissions**         | ``string``                   | The groups that have permissions to view and use this space template.      |
++-------------------------+------------------------------+----------------------------------------------------------------------------+
+| **homePageApplication** | ``SpaceApplication``         | The application to use in the home page of a space.                        |
 +-------------------------+------------------------------+----------------------------------------------------------------------------+
 | **applications**        | list of ``SpaceApplication`` | The list of the applications that are installed by default to a new space. |
 +-------------------------+------------------------------+----------------------------------------------------------------------------+
 
+.. tip:: The ``permissions`` parameter is a list of groups separated by ";" character.
```

Comment:
member
```suggestion
.. tip:: The ``permissions`` parameter is a list of membership expressions separated by ";" character.
```

Reply:
Does it accept simple group expressions without ":" ? such as "/platform/users"

Reply:
No, it must fit the format MEMBERSHIP:GROUP_ID